src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => activeTabId === tab.id) || tabs[0];
+
+  const handleTabClick = tab => {
+    if (activeTab.id !== tab.id) {
+      onTabSelected(tab.id);
+    }
+  };
```

Comment:
And this logic I would pass to the parent component. Just as a variant)

Reply:
Yep. The best way is that the component returns a non non-validated value ( without condition) and the parent component makes all preparation or validation with this data. 
It`s better for reusability. Mabe in another patent component we don't need this condition)